From 1b1dfef1529814ddf87f4dc0ca95e8463ebae47c Mon Sep 17 00:00:00 2001
From: Jonathan Perkin <jperkin@joyent.com>
Date: Wed, 3 Aug 2016 17:40:09 +0100
Subject: [PATCH] OS-5568 joyent manifest should include nis/client

Reviewed by: Robert Mustacchi <rm@joyent.com>
Approved by: Patrick Mooney <patrick.mooney@joyent.com>
---
 manifest | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/manifest b/manifest
index febbc61e97..f611be280e 100644
--- a/manifest
+++ b/manifest
@@ -1542,6 +1542,8 @@ f lib/svc/manifest/network/nfs/mapid.xml 0444 root sys
 f lib/svc/manifest/network/nfs/nlockmgr.xml 0444 root sys
 f lib/svc/manifest/network/nfs/server.xml 0444 root sys
 f lib/svc/manifest/network/nfs/status.xml 0444 root sys
+d lib/svc/manifest/network/nis 0755 root sys
+f lib/svc/manifest/network/nis/client.xml 0444 root sys
 f lib/svc/manifest/network/npiv_config.xml 0444 root sys
 f lib/svc/manifest/network/rarp.xml 0444 root sys
 f lib/svc/manifest/network/rexec.xml 0444 root sys
-- 
2.21.0

